public_html/rpcs4.php

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html lang="en-GB">
+<head>
+<title>RPCS4 - PlayStation 4 Emulator</title>
+<meta charset="utf-8">
+<meta name="description" content="RPCS3 is a multi-platform open-source Sony PlayStation 3 emulator and debugger written in C++ for Windows, Linux, macOS and FreeBSD.">
```

Comment:
Rpcs3 ?
PlayStation 3?

Reply:
It's a redirect, so the meta tags don't really matter, it's just mimicking the ones from the normal pages